### ISSUE TITLE
"?" should not count as operator character when followed by alphanumeric

### ISSUE THREAD

**Issue by jonleivent** [cpitclaudel/company-coq#37]
When a ? is followed immediately (no whitespace in between) by an alphanumeric, I believe that Coq in 8.5 has to parse the result is an evar (or pattern binding, or rewrite scheme, or intropattern, but these don't show up in the goals window).  If so, this should be taken into account with character folding.  I have a prefix operator symbol that I'm trying to fold, but when it operates on an evar, it's not folding due to the ?.

**Comment by cpitclaudel** [comment]
Sorry for not getting to this earlier. Can you give me a slightly more concrete example of the problem? If it with something like `?123<=?456` where `<=` is not being prettified?

**Comment by jonleivent** [comment]
Yes - with operators that don't have spaces around them, and that appear in the goals window in front of evars.  So, something like x<=?y, the <= is not prettified.  The particular case I have is of a unary prefix operator being prettified, but I guess it would happen with binary operators as well, whenever Coq doesn't put a space in between them an the second operand.
